shared/src/commonMain/kotlin/io.newm.shared/repositories/CardanoWalletRepository.kt

```diff
@@ -33,15 +34,10 @@ fun List<LedgerAssetMetadata>.getMusicMetadataVersion(): Int? {
 }
 
 data class NFTTrack(
-    val name: String,
-    val image: String,
-    val imageMimeType: String,
-    val files: List<File>,
-    val musicMetadataVersion: Int,
-    val releaseType: String?,
-    val releaseTitle: String?,
-    val distributor: String?,
-    val publicationDate: String?
+    val songName: String,
```

Comment:
Should this be `trackName` to be consistent with the class type?  In which case `name` would be fine to avoid redundency.